cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -96,14 +97,25 @@ public void tearDown() throws Exception {
     // TODO: check metrics, streams, etc.
   }
 
-  protected TestManager getTestManager() {
+  protected ClientConfig getModifiableClientConfig(ClientConfig initialClientConfig, Id.Namespace namespace) {
```

Comment:
and how about it internally calls `getClientConfig`, instead of requiring user to pass it in?
Maybe call it `getClientConfig(Id.Namespace)`